### DISCUSSION TITLE
How do I edit the torrent.rc file?

### DISCUSSION THREAD

**Discussion by CraftyClown** [crazy-max/docker-rtorrent-rutorrent#283] [General]
Hi folks, sorry if this is a silly question but I'd like to edit the torrent.rc file and I'm not sure how to access and edit it in this container.
I need to change the folder paths from the standard downloads/complete so that I can get hard linking working with Radarr and Sonarr and I'm presuming the only way is via the .rc file
any assistance would be greatly appreciated

**Comment by kerryandjane** [comment]
Try this.
Create or edit /data/rtorrent/.rtorrent.rc and restart. 

To do this you have a few options:
1. Map the data folder in docker/docker compose (easy)
2. `docker exec -it rtorrent sh` (less easy)

My advice is to keep it all simple.

Personally I use this line in my rtorrent.rc file.
`method.set_key = event.download.finished,spit_out,"execute=/scripts/flexgot.sh,complete,$d.custom1=,$d.name=,$d.get_finished_dir=,$d.timestamp.started=,$d.timestamp.finished=,$d.size_bytes=,$d.up.total="`

After a download is completed it pulls out alot of infomation from the rtorrent and runs a script that does all the work (unrar, moves it to the right directory based on what it's $d.custom1= aka "label" in rutorrent, notifications etc.)

Something like this could work if you could could ensure the label in rutorrent from radarr and sonarr are set to the directory path /volume1/sonarr/..... 
`method.insert = d.get_finished_dir, simple, "cat=$cfg.download_complete=,$d.custom1="
method.insert = d.copy_to_complete, simple, "d.directory.set=$argument.1=; execute=mkdir,-p,$argument.1=; execute=cp,-u,$argument.0=,$argument.1=; d.save_full_session="
method.set_key = event.download.finished,copy_complete,"d.copy_to_complete=$d.data_path=,$d.get_finished_dir="
`
Basic scripting outside of rtorrent is my choice though.... easier to test for me.
